ext/pom.xml

```diff
@@ -12,53 +13,56 @@
 		of the GNU Lesser General Public License, v.2.1 along with this distribution; 
 		if not, write to the Free Software Foundation, Inc., 51 Franklin Street, 
 		Fifth Floor, Boston, MA 02110-1301, USA. (C) 2009, @author JBoss Inc. (jonathan.halliday@redhat.com) -->
-    <parent>
-        <groupId>org.jboss.narayana</groupId>
-        <artifactId>narayana-all</artifactId>
-        <version>5.12.8.Final-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.jboss.narayana.ext</groupId>
-    <artifactId>ext-all</artifactId>
-    <packaging>pom</packaging>
-    <name>Narayana: ext</name>
-    <description>Narayana: Ext All</description>
+  <parent>
+    <groupId>org.jboss.narayana</groupId>
+    <artifactId>narayana-all</artifactId>
+    <version>5.12.8.Final-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <groupId>org.jboss.narayana.ext</groupId>
+  <artifactId>ext-all</artifactId>
+  <packaging>pom</packaging>
+  <name>Narayana: ext</name>
+  <description>Narayana: Ext All</description>
 
-    <properties>
-        <sortpom.skip>true</sortpom.skip>
-    </properties>
+  <properties>
+    <sortpom.skip>false</sortpom.skip>
+  </properties>
```

Comment:
Thanks for applying sortpom to this module +1
Instead of switching sortpom off (i.e. `sortpom.skip = false`), just remove this property

Reply:
lol, I forgot, sortpom is more of a hindrance than and an aid.

Reply:
I only turned it on because I couldn't be bothered to format it manually since I had done a fair bit of cut and pasting into the pom.

Reply:
And it makes it hard to see what the PR actually changed.

Reply:
Although in this PR I only changed the plugin so it should be easy for you to see the functional changes.

Reply:
> Although in this PR I only changed the plugin so it should be easy for you to see the functional changes.

+1

Reply:
Could you please remove this property from the pom? Sortpom is activated by default in the [main pom](https://github.com/jbosstm/narayana/blob/master/pom.xml#L187-L188)